tools/db_crashtest.py

```diff
@@ -356,6 +373,12 @@ def gen_cmd_params(args):
     if args.test_best_efforts_recovery:
         params.update(best_efforts_recovery_params)
 
+    # Best-effort recovery and BlobDB are currently incompatible. Test BE recovery
+    # if specified on the command line; otherwise, apply BlobDB related overrides
+    # with a 10% chance.
```

Comment:
Is it worth having a BlobDB only stress test job?

Reply:
That's what I'm trying to avoid :) Note that BlobDB should also work with features like transactions and atomic flush, so testing it by itself as well as in conjunction with those features would require setting up jobs for the direct product `{no sanitizer, ASAN, TSAN, UBSAN} x {no additional features, atomic flush, transactions}`, which is 12 jobs. If you toss in `{whitebox, blackbox}` as well, it's 24 jobs (and it might be more later).